db/data.py

```diff
@@ -58,6 +58,44 @@ def _has_key(dictionary, key):
         dictionary = dictionary[part]
     return True
 
+def check_highlevel_meta():
+    with db.engine.begin() as connection:
+        query = text("""
+                        SELECT COUNT(*)
+                        FROM highlevel_meta ;
```

Comment:
there's a better way of performing this query - instead of doing one query for highlevel and one for highlevel_meta, we can do a join and just count the missing rows:
`select id from highlevel left join highlevel_meta using (id) where highlevel_meta.id is null`

Double-check the formatting of other queries in this file. We need to right-align keywords.

Queries run via sqlalchemy shouldn't have a ; at the end of them.